app/routes/regulatory-attachments/index.js

```diff
@@ -30,12 +30,12 @@ export default class ListRoute extends Route {
         number: params.page,
         size: params.size,
       },
+      include: 'published-version.current-version',
```

Comment:
please add a comment here to explain what this include is doing, cause includes should only ever be for optimization, but in this case it's keeping the app together

here's what I wrote in rocket chat, but please reword it a bit

It's basically ED doesn't know about the update, so it doesn't think it has to refetch

however, query always refetches, it never caches, so when we include that data in the query, we force it to update